source/Projectile.cpp

```diff
@@ -73,17 +73,19 @@ Projectile::Projectile(const Projectile &parent, const Point &offset, const Angl
 	if(inaccuracy)
 	{
 		this->angle += Angle::Random(inaccuracy) - Angle::Random(inaccuracy);
-		if(!parent.weapon->Acceleration())
-		{
-			// Move in this new direction at the same velocity.
-			// To maintain the sign of the velocity, Point::Length can’t be used.
-			Point referenceVector = parent.angle.Unit();
-			double parentVelocity = referenceVector.Dot(parent.velocity);
-			velocity = this->angle.Unit() * parentVelocity;
-		}
 	}
-	velocity += this->angle.Unit() * (weapon->Velocity() + Random::Real() * weapon->RandomVelocity());
 
+	// Start with the parent's current velocity
+	velocity = parent.velocity;
```

Comment:
This is redundant; the velocity is already equal to the parent's velocity.

Reply:
I'm aware of that but I don't believe any random velocity value is carried through from firing.  Even still; we could use half of the random velocity stat instead of randomizing it to the full value like I've done.  Or we could ignore it but I wouldn't do that.

Reply:
Thanks for catching that.  I removed the assignment.